### ISSUE TITLE
Add a "What's New" section to splash screen

### ISSUE THREAD

**Issue by jwhaney** [TNRIS/lake-gallery#31]
"Users will be able to see what new info/reservoirs/features are available."

**Comment by jwhaney** [comment]
@csidenblad & @LaurenKirk, I think we talked about this one but do not think we came to a conclusion on this. Need some clarification & specifics to move forward.

If by 'splash screen' you mean the modal that shows up when i user first visits the app, or that continues to show unless they check the checkbox saying don't show this again, then I think I disagree with the location of this feature/enhancement. I think the modal should only show a welcome screen and a description about what the app is, along with instructions for new users. Something like a 'What's new' section should be something that can be accessed/visible whenever a user wants. The modal that appears now, if dismissed by the checkbox, will not show up again unless the user knows how to clear their local storage. That of course means the user would never see the 'What's new' section again.

That being said, I don't know where the location for a What's new section should go, or if it really provides that much use/information. I think my opinion is that this feature shouldn't be implemented.
